api4/team.go

```diff
@@ -235,6 +236,60 @@ func getTeamMembersByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.TeamMembersToJson(members)))
 }
 
+func addTeamMember(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	var tm *model.TeamMember
+	var err *model.AppError
+	var team *model.Team
+	if team, err = app.GetTeam(c.Params.TeamId); err != nil {
+		c.Err = err
+		return
+	}
```

Comment:
noted. for the request body, i've implemented this,
```requestBody := map[string]string{"team_id": teamId, "user_id": userId}``` 
(i just assumed that is the convention here, just let me know if I should change). thanks